docs/releases.rst

```diff
@@ -16,6 +16,12 @@ results are represented as ``<testsuite>`` tags, each containing a
 ``<testcase>`` tag for the main result, along with additional
 ``<testcase>`` tags for any subresults.
 
+A new ``tmt-ensure-efiboot-order`` script has been added to help manage
```

Comment:
What about using a bit more generic name such as `tmt-boot-order` or `tmt-set-boot-order`? Not sure if there are (or will be) more similar use cases, but the currently proposed name sounds very specific to `efiboot`. Perhaps, in the future, we would like to support not only preserving the current one but setting a specific one? Just brainstorming...